src/main/java/io/cdap/delta/datastream/DatastreamTableRegistry.java

```diff
@@ -192,7 +192,7 @@ public void close() throws IOException {
   private DiscoverConnectionProfileRequest.Builder buildDiscoverConnectionProfileRequest(
     String sourceConnectionProfileName) throws IOException {
     DiscoverConnectionProfileRequest.Builder request =
-      DiscoverConnectionProfileRequest.newBuilder().setParent(parentPath).setFullHierarchy(true);
+      DiscoverConnectionProfileRequest.newBuilder().setParent(parentPath).setHierarchyDepth(2);
```

Comment:
we should add comment for meaning of hierarchy depth and rationale for setting it to 2.

Reply:
Current unit test is for describe table method, thus has no code updates.Unit tests pass.